scripts/validate_docstrings.py

```diff
@@ -236,6 +237,15 @@ def parameter_type(self, param):
     def parameter_desc(self, param):
         return self.doc_parameters[param][1]
 
+    def parameter_desc_without_directives(self, param):
```

Comment:
What happens if you just use this logic in `parameter_desc` instead of as a separate method? I can't think of a case where we want the directives to be considered part of the description, and the way you've coded this works well for the missing period but may not be generalizable to other issues that could come up

Reply:
That's a good point. I'd probably keep a `raw_parameter_desc` with what `parameter_desc` has now, as in the future I'd probably like to have the values of the directives (for example to find the deprecated parameters we need to delete in the next version).